packages/vscode-extension/src/uriHandler.ts

```diff
@@ -4,6 +4,11 @@ import * as queryString from "query-string";
 import { localize } from "./utils/localizeUtils";
 import * as util from "util";
 import { SwitchToPreReleaseVersionLink } from "./constants";
+import { EventEmitter, Uri } from "vscode";
+
+export namespace ext {
```

Comment:
There's already `globalVariables.ts` for common variables used throughout the extension. Better to add `uriEventHandler` there.

Reply:
Done.